sysutils/ntpstat/files/1001-Makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- Makefile.orig	2013-05-23 16:15:41.000000000 -0400
++++ Makefile	2021-12-05 11:50:05.000000000 -0500
+@@ -1,5 +1,6 @@
+-CC= gcc
+-CFLAGS= -Wall -O2
++CC = gcc
```

Comment:
how does this make sure that the right compiler will be used - is the MacPorts' selected compiler used? Did you run the test described in the link I mentioned earlier to make sure it works? (I haven't but can probably do that tomorrow or so if you cannot figure out how to do it.

Reply:
okay, just tried it and that actually works fine. I though one should also add the `?` for the compiler as in `CC ?= gcc`, but that doesn't seem to be needed